global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "2.1.300"
+    "version": "2.1.403"
```

Comment:
why is this updated?

Reply:
It shouldn't. It's just a fix for me to run it. I'll change it back now.